sites/funcampus/src/backend/templates/courses/cms/course_run_detail.html

```diff
@@ -0,0 +1,4 @@
+{% extends "courses/cms/course_run_detail.html" %}
```

Comment:
Is it relevant to override `course_run_detail` template?

Reply:
Yes it is. It is a mirror of the course_detail.html template. In a future refactoring I will remove all course run pages because we realized it is not how course runs will be handled IRL. Having to maintain a page for each course run is painful and useless... This will also remove the complexity that you are pointing here...